dataformats/multigrid/inc/RunSpecParseTest.cpp

```diff
@@ -0,0 +1,53 @@
+/** Copyright (C) 2016 European Spallation Source ERIC */
+
+#include <prototype2/common/DataSave.h>
+#include <RunSpecParse.h>
+#include <test/TestBase.h>
+#include <cstring>
+
+
+std::string filename{"deleteme.json"};
+
+std::string runspecjson = " \
+{ \n\
+  \"test\" :\n\
+    [ \
+      {\"id\": 42,  \"dir\": \"mydir\",  \"prefix\": \"myprefix\",  \"postfix\": \"mypostfix\", \"start\": 1, \"end\":   999, \"thresh\": 124} \n\
```

Comment:
As pointed out by someone during a previous code review, C++ now supports string literals (https://en.cppreference.com/w/cpp/language/string_literal). I suggest that you replace this string with that to make it more readable.